tollbros/tollbrothers-ui
# Run eslint/prettier & add PR action for linting

## JIRA Ticket
[NJP-2193](https://tbnextjs.atlassian.net/browse/NJP-2193)

## Describe Changes in this Pull Request
- Ran eslint via `npm run test:lint -- --fix`.
- Updated nvmrc and node engine in package json
- Updated prettierrc
- Created a PR github action so linting is ran on every PR


## Testing Instructions
- Verify no regression exists for components. I had to make some heavy handed changes to the Mortgage Calculator.

## Screen Shot(s)/Video(s) of Working Changes
No UI changes.

### Tasks:

1. [ ] Changes Meet JIRA Tickets Requirements
2. [ ] Commit message with `fix` or `feat` to trigger new release
3. [ ] Tested changes on Mobile, Tablet and Desktop screen sizes
4. [ ] Tested changes on Chrome, FireFox and Safari
5. [ ] Demoed to Design Team (For new features only)
6. [ ] Notified Digital Marketing Team if changes will impact A/B Tests
7. [ ] Checked Tracking/Analytics (Notified Digital Marketing Team if changes impact tracking)
